gravitee-apim-console-webui/src/management/api/creation-v4/api-creation-v4-spec-http-expects.ts

```diff
@@ -33,13 +33,14 @@ import { RestrictedDomain } from '../../../entities/restricted-domain/restricted
 export class ApiCreationV4SpecHttpExpects {
   constructor(private httpTestingController: HttpTestingController) {}
 
-  expectApiGetPortalConfiguration() {
+  expectApiGetPortalConfiguration(portalConfiguration?: PortalConfiguration) {
     const portalConfig: PortalConfiguration = {
       portal: {
         entrypoint: 'entrypoint',
         kafkaDomain: 'kafka.domain',
         kafkaPort: 9092,
       },
+      ...portalConfiguration,
```

Comment:
```suggestion
    const portalConfig: PortalConfiguration = {
      ...portalConfiguration,
      portal: {
        entrypoint: 'entrypoint',
        kafkaDomain: 'kafka.domain',
        kafkaPort: 9092,
      },
```
WDYT to avoid overwriting the `portal` object information?

Reply:
Hum it's the purpose to be able to override this portal a bit like a fakeSomething. we have default value that can bet overwrited if needed. 
No ?